client/boot/index.js

```diff
@@ -186,7 +186,7 @@ function boot() {
 		analytics.setSuperProps( superProps );
 
 		if ( config.isEnabled( 'oauth' ) ) {
-			LoggedOutLayout = require( 'layout/logged-out-oauth' );
+			LoggedOutLayout = require( 'layout/logged-out' );
```

Comment:
We need to display two different layout styles before the user logs in, based on the whether or not the user is on `/login` or `/start`. The only way I can think to accomplish this is to update `LoggedOutLayout` to display both styles based on whether or not `props.section === 'signup'`.

This could be wrong, but I don't believe we can use two separate layouts here, because switching would wipe out `#primary`, which is what we render the other faux-top-level React components into.